python/processor.py

```
@@ -352,10 +354,126 @@ def process_command(action: str, args: Dict[str, str]) -> Dict[str, Any]:
+            result_data = processor.separate_stems(args['input_path'], args['output_dir'])
+            response = format_json_response('success', {
+                'stems': result_data,
+                'details': {
+                    'input_file': args['input_path'],
+                    'output_directory': args['output_dir']
+                }
+            })
+
+    except Exception as e:
+        # Catch exceptions from processor.separate_stems or other issues
+        logger.error(f"Error during stem separation: {str(e)}", exc_info=True)
```

Ruff (0.14.3)

Use explicit conversion flag

Replace with conversion flag

(RUF010)